enterprise/backup/src/main/java/org/neo4j/backup/BackupService.java

```diff
@@ -219,6 +223,10 @@ public void configure( Map<String, String> config )
                 targetDb.shutdown();
             }
             bumpMessagesDotLogFile( targetDirectory, backupStartTime );
+            if ( config.get( GraphDatabaseSettings.rebuild_idgenerators_fast ) )
```

Comment:
I'm not sure about re-using this config option with this new purpose. Even if it is closely related.